research/sentiment_analysis/sentiment_model.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+def _dynamic_pooling(w_embs):
+    import tensorflow as tf
```

Comment:
why import tf again?

Reply:
I have encountered this in the past. If you save a keras model with custom functions and then load them something can get confused in the imports. So importing inside the function is a common way to get around that. (Not sure if that's the case here.)

Reply:
yes that was the case.